dev-requirements.txt

```diff
@@ -6,6 +6,8 @@ isort~=5.7
 pytest~=5.4.3
 coverage~=7.5
 pytest-benchmark~=3.2.3
+# Note: you might need a Lua installation to install lupa on Mac.
```

Comment:
might, or will?

Reply:
At the moment Lua is needed, since the relevant version has no macOS&Arm distribution:
https://pypi.org/project/lupa/1.14.1/#files
When `lupa` is upgraded to `>=2.2` - there will be no need, as there's a dedicated dist. for the mentioned platform:
https://pypi.org/project/lupa/2.2/#files

But I think the upgrade should be done separately.

Reply:
By the way, do you happen to know what it is even used for? I didn't see any usage of it in the project